.github/renovate.json

```diff
@@ -5,7 +5,7 @@
     ":rebaseStalePrs",
     "group:linters",
     "group:test",
-    ":preserveSemverRanges"
+    ":preserveSemverRanges",
```

Comment:
I don't believe you can use trailing commas in JSON and I'd rather be safe than sorry so lets' revert this change 👍